File: task2/src/Entities/Book.java
import java.util.UUID;

public class Book {
    public UUID id;
    public String title;
    public UUID authorId;
    public String shortDescription;
    public Double cost;

    public Book() {

    }

    public Book(UUID id, String title, UUID authorId, 
                    String shortDescription, Double cost) {
        this.id = id;
        this.title = title;
        this.authorId = authorId;
        this.shortDescription = shortDescription;
        this.cost = cost;
    }
}

File: libraryAngular/src/app/entities/book.ts
import {Author} from './author'
import {Genre} from './genre'

export class Book {
    id: string = "";
    title: string = "";
    author: Author = new Author();
    shortDescription: string = "";
    cost: number = 0;
    downloadLink: string = "";
    imageLink: string = "";
    genres: Genre[] = []
}